ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -44,7 +44,7 @@ StatusDialog {
         }
 
         function fetchSuggestedRoutes() {
-            if (payPanel.valueValid && root.swapInputParamsForm.isFormFilledCorrectly()) {
```

Comment:
hmm wouldn't this not catch some cases where we are indeed interested in not refetching, like not valid numbers and that sort of thing?
Perhaps we need to discriminate two levels of check:
- ValidNumber (whatever string the user entered can be converted to a number)
- ValidValue (the number entered by the user passes some checks)

Reply:
Hey @dlipicar we do a 
1. not a number
2.  not an empty string  &
3. value > 0

check in the form and I think thats enough as a check :)

Reply:
The fix looks good for now if we're making the checks externally. However I feel the base component would be more useful if it had the 2 signals (`validNumber` and `validValue`) separately, that way we could reuse that part of the logic instead. @caybro do you think that's reasonable? not for this PR, but for some future improvement, of course.

Reply:
yeah, more like bindings so that we can check whether the number and value is valid